Scripts/BulkConversion/bulk_convert.ps1

```diff
@@ -0,0 +1,655 @@
+. $psscriptroot\SharedScriptLib.ps1
+
+function CreateMPTTemplate([Parameter(Mandatory=$True,ParameterSetName=$('VirtualMachine'),Position=0)]$virtualMachine, 
```

Comment:
Can multiple params have same position?

Reply:
Yes, as long as the parameters are part of different Parameter Sets.

Reply:
Yup, as long as the parameters are part of different parameter sets.